assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -395,6 +395,9 @@ che.infra.openshift.username=
 che.infra.openshift.password=
 che.infra.openshift.trust_certs=
 
+# Create routes with TLS enabled
+che.infra.openshift.enable_tls=
```

Comment:
It would be better to set default value here

Reply:
sounds good